packages/render-fidelity-tools/test/renderers/filament/filament/docs/filamesh.md

```diff
@@ -0,0 +1,264 @@
+# Filamesh
```

Comment:
I see a bunch of filament files here, just wondering why, is this intended?

Reply:
I didn't realize I shouldn't add those. Will revert and send a new iteration